sorter/src/main/java/sortpom/SortPomImpl.java

```diff
@@ -77,6 +77,12 @@ private void warnAboutDeprecatedArguments(SortPomLogger log, PluginParameters pl
         if (pluginParameters.sortDependencies.isDeprecatedValueFalse()) {
             log.warn("[DEPRECATED] The 'false' value in sortDependencies is not used anymore, please use empty value '' or omit sortDependencies instead. In the next major version 'true' or 'false' will cause an error!");
         }
+        if (pluginParameters.sortExclusions.isDeprecatedValueTrue()) {
+            log.warn("[DEPRECATED] The 'true' value in sortExclusions is not used anymore, please use value 'groupId,artifactId' instead. In the next major version 'true' or 'false' will cause an error!");
+        }
+        if (pluginParameters.sortExclusions.isDeprecatedValueFalse()) {
+            log.warn("[DEPRECATED] The 'false' value in sortExclusions is not used anymore, please use empty value '' or omit sortDependencies instead. In the next major version 'true' or 'false' will cause an error!");
+        }
```

Comment:
I wasn't sure about this. One alternative is e.g. to directly throw an exception for this case in `AbstractParentMojo#execute`, but then the `true`/`false` handing is spread over multiple places.

Reply:
Hmmm, it is not right, but the alternative is worse. I guess that we could create a subclass structure for PluginParameters and that would just complicate the code. We let it be for now.

Reply:
Now I know, true or false in sortDependencyExclusions will fail the build instead